src/worker/compile_wasm_no_simd.sh

```diff
@@ -1 +1 @@
-em++ -std=c++11 sorter_no_simd.cpp -Os -s WASM=1 -s SIDE_MODULE=2 -o sorter_no_simd.wasm -s IMPORTED_MEMORY=1 -s SHARED_MEMORY=1
\ No newline at end of file
+em++ -std=c++11 sorter_no_simd.cpp -Os -s WASM=1 -s SIDE_MODULE=2 -o sorter_no_simd.wasm -s IMPORTED_MEMORY=1 -s USE_PTHREADS=1
```

Comment:
was it some troubleshooting changes? is that needed? 

I would not touch this given the no_simd is likely a script for compatibility sake. Not sure tbh.